src/main/resources/config.yml

```diff
@@ -174,7 +174,23 @@ old-golden-apples:
   enabled: true
   worlds: []
   # Cooldown between eating the apples, in seconds
-  cooldown: 0
+  cooldown:
+    # The cooldown for normal golden apples
+    normal: 0
+    # The cooldown for enchanted golden apples
+    enchanted: 0
+    # Whether the two apple types share a cooldown.
+    # If this is true:
+    #   1. Eating any apple resets the cooldown time
+    #   2. Each apple type can only be eaten when its cooldown time os over
+    #      This means, that when you eat *any* apple you start two parallel cooldowns: One for enchanted and one
+    #      for normal apples. Each type can be eaten when its cooldown is over.
+    #      Once any apple was eaten, the parallel cooldowns are *both* restarted, so you can not eat either type again
+    #      before its full cooldown is over.
```

Comment:
Personally, I would shorten this explanation and simply state that if the cooldown is shared, that means enchanted and normal gapples are treated as the same item for purposes of the cooldown.

Reply:
The length of the cooldown depends on what item you are trying to consume *after* the cooldown started, not on the item you consumed to *start* the cooldown. I just wanted to make that clear, but I am open for rephrasings.

Reply:
Hmm, is that because of the way the timers are implemented? Surely, to the end user that results in the same effect as just saying that they're the same?

Reply:
There are two possible behaviours a "shared" cooldown could be:
1. The initially consumed item counts
  - You consume an enchanted apple
  - You can no longer consume *any* apple until the *enchanted* cooldown is over
  - You consume a normal apple
  - You can no longer consume *any* apple until the *normal* cooldown is over

2. The later consumed item counts
  - You consume an enchanted *or normal* apple
  - You can no longer consume any *enchanted* apple until the *enchanted* cooldown is over
  - You can no longer consume any normal apple until the *normal* cooldown is over

The implemented cooldown uses version 2.

Reply:
What I intend by _shared_ is what the plugin was already doing before this patch. Which of those two options would you say matches the previous behaviour?

Reply:
Both, if you set both cooldowns to the same value. If you have different values for both, it is necessarily different than before.

Reply:
Aaah so you're saying that the different timer values are kept even when in shared mode!

I see, if that's the case, then I would be inclined to rephrase the description with something like this:

```
 # If this is true:
    #   1. Eating any apple resets both cooldowns
    #   2. Each apple type can only be eaten when its cooldown time is over
    #      This means that when you eat *any* apple you start two parallel cooldowns: One for enchanted and one
    #      for normal apples. Each type can only be eaten when its cooldown is over.
    #      Once any apple is eaten, both cooldowns are restarted, so you can not eat either type again
    #      before its full cooldown is over.
    #   3. To have the plugin treat normal and enchanted golden apples as having the same cooldown,
    #      then set the same cooldown time and enable shared mode. (This was the old mode)
```

Reply:
Done.